src/tools/hash-text/hash-text.vue

```diff
@@ -44,11 +31,11 @@ const algos = {
   RIPEMD160,
 } as const;
 
+type AlgoNames = keyof typeof algos;
+const algoNames = Object.keys(algos) as AlgoNames[];
+
 const clearText = ref(
-  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lacus metus blandit dolor lacus natoque ad fusce aliquam velit.',
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lacus metus blandit dolor lacus natoque ad fusce aliquam velit.'
 );
-const algo = ref<keyof typeof algos>('SHA256');
-const hashedText = computed(() => algos[algo.value](clearText.value).toString());
-
-const { copy } = useCopy({ source: hashedText, text: 'Hash copied to the clipboard' });
+const hashText = (algo: AlgoNames, value: string) => (value ? algos[algo](value).toString() : '');
```

Comment:
Let me know if you think an empty string should be hashed anyway.

Reply:
I think an empty string should be hashed, it makes sens 

cf:
https://crypto.stackexchange.com/questions/26133/sha-256-hash-of-null-input
https://www.di-mgt.com.au/sha_testvectors.html

Reply:
fair point